pyproject.toml

```
@@ -1,3 +1,80 @@
+[project]
+name = "pyfar"
+version = "0.7.0"
+description = "Project for data formats in acoustics."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.9"
+authors = [
+    { name = "The pyfar developers", email = "info@pyfar.org" },
+]
+keywords = [
+    "pyfar",
```

```suggestion
    "pyfar",
    "signal processing",
    "signal visualization",
    "Fourier transforms",
```
please comment or add more :)

---

- I would use 'audio signal' instead of 'signal'
- I would add:
  - Filterbanks
  - Parametric filters
  - Read and write audio signals